src/pages/List.js

```diff
@@ -38,19 +69,20 @@ const List = props => {
         // Sort the data
         sort="name"
         // Only fetch the items associated with the token saved in localStorage
-        filter={['token', '==', token || getCurrentToken() || 'no token set']}
```

Comment:
Careful here @MonicaDJohnson. This is a breaking change when the user doesn't already have a token saved in `localStorage`. I know this is from the branch this PR is based on, but just a heads up.